types/backbone/index.d.ts

```diff
@@ -314,6 +322,14 @@ declare namespace Backbone {
         take(): TModel;
         take(n: number): TModel[];
         toArray(): TModel[];
+
+        /**
+         * Sets the url property (or function) on a collection to reference its location on the server.
+         *
+         * @memberof Collection
+         */
+        url: string | Func<string>;
```

Comment:
You can just use `string | (() => string)`, right?

Reply:
Yeah, I had tried that but I was missing the outer parentheses causing the tests to fail.  Thanks for the feedback and I'll rebase this pull request.